lib/telemetry_metrics_statsd/event_handler.ex

```diff
@@ -52,7 +55,7 @@ defmodule TelemetryMetricsStatsd.EventHandler do
             # The order of tags needs to be preserved so that the final metric name is built correctly.
             tag_values = metric.tag_values.(metadata)
             tags = Enum.map(metric.tags, &{&1, Map.fetch!(tag_values, &1)})
-            Formatter.format(prefix, metric, value, tags)
+            formatter_mod.format(prefix, metric, value, tags)
```

Comment:
I was wondering about how we could handle `:prefix`. Since it's a global setting, I thought that it could be handled outside of formatter scope. Whatever the formatter returns, we add a given prefix. Alternatively, it could be passed as an extra formatter option (similar to Logger formatters) but this way we would need to implement the whole formatter machinery, which might not be needed.

What do you think?